tests/models/bart/test_modeling_bart.py

```diff
@@ -599,13 +599,15 @@ def test_xsum_1_1_generation(self):
             " 2002 to prosecute genocide, crimes against humanity and war crimes."
         )
         EXPECTED = (
+            "</s>"
             " The International Criminal Court (ICC) has announced that it has been announced by the International"
             " Criminal court."
+            "</s>"
         )
 
         dct = tok(ARTICLE, return_tensors="pt")
         generated_ids = hf.generate(**dct, num_beams=4)
-        result = tok.batch_decode(generated_ids, skip_special_tokens=True)[0]
+        result = tok.batch_decode(generated_ids)[0]
```

Comment:
Tests: update beam search tests to also print special tokens

e.g. this updated test fails on main because it it's outputting extra pad tokens